service-registry-cli/README.adoc

```diff
@@ -0,0 +1,327 @@
+
+.Prerequisites
+
+* You are logged in to `rhoas`.
+
+.Procedure
+
+. Enter the following command to create an instance with default configuration.
++
+--
+This example creates a Service Registry instance called `my-registry`.
+
+.Creating a Service Registry instance
+[source,shell]
+----
+$ rhoas service-registry create --name my-registry
+Creating Service Registry instance with name: my-registry
+✔️ Successfully created Service Registry instance
+{
+  "browserUrl": "https://console.redhat.com/beta/application-services/service-registry/t/63620903-4a29-4ba4-8229-7ab793bf73f3",
+  "created_at": "2021-09-27T15:35:01Z",
+  "href": "/api/serviceregistry_mgmt/v1/registries/63620903-4a29-4ba4-8229-7ab793bf73f3",
+  "id": "63620903-4a29-4ba4-8229-7ab793bf73f3",
+  "instance_type": "eval",
+  "kind": "ServiceRegistry",
+  "name": "my-registry",
+  "owner": "wtrocki@redhat.com",
+  "status": "accepted",
+  "updated_at": "2021-09-27T15:35:01Z"
+}
+----
+
+[NOTE]
+====
+After creation your service registry instance will be added to service context and used by default
+when executing other instance specific commands
+====
+--
+
+. Enter the following command to verify that the instance is ready to use.
++
+--
+This command shows that the Service Registry instance is ready to use,
+because the `Status` field is `ready`.
+
+.Reviewing details of a Registry instance
+[source,shell]
+----
+$ rhoas status service-registry
+
+  Service Registry
+  -----------------------------------------------------------------------------------------------------------------------------------------------------------------
+  ID:                    63620903-4a29-4ba4-8229-7ab793bf73f3
+  Name:                  my-registry
+  Status:                ready
+  Registry URL:          https://service-registry-service-registry-production.apps.app-sre-prod-04.i5h0.p1.openshiftapps.com/t/63620903-4a29-4ba4-8229-7ab793bf73f3
+----
+
+[NOTE]
+====
+When you created the Service Registry instance, it was set as the current instance automatically.
+If you have multiple Service Registry instances,
+you can switch to a different instance by using the `rhoas service-registy use` command.
+====
+--
+
+[discrete,id="creating-service-account_registry_{context}"]
+=== Creating a service account for Service Registry
+
+To connect your applications or services to a Service Registry instance, you must first create a service account with credentials.
+The credentials are exported to a file on your computer,
+which you can use to authenticate your application with your Kafka instance.
+
+.Prerequisites
+
+* You have created a Service Registry instance, and it has a `ready` status.
+
+.Procedure
+
+. Enter the following command to create a service account.
++
+--
+This example creates a service account called `my-service-acct` and saves the credentials in a JSON file.
+
+.Creating a service account
+[source,shell]
+----
+$ rhoas service-account create --name "my-service-acct" --file-format json
```

Comment:
@bibryam @ErikWittman
Should we build guide for current authorization policies (no need to apply specific roles to use service registry) or roles that are required in QA?